test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -237,6 +237,19 @@ protected virtual DocumentStore GetDocumentStore(Options options = null, [Caller
                                 await WaitForRaftIndexToBeAppliedInCluster(result.RaftCommandIndex, timeout);
                             });
                         }
+
+                        // skip 'wait for requests' on DocumentDatabase dispose
+                        Servers.ForEach(x =>
```

Comment:
Why?

Reply:
On store dispose we send command to delete the db, and then WaitForIndexNotification of that command, which has a timeout of 15 sec, but the DB dispose has '[drain requests code](https://github.com/ravendb/ravendb/blob/v4.2/src/Raven.Server/Documents/DocumentDatabase.cs#L633)' which can take up to 60 secs, and make the WaitForIndexNotification throw timeout exception and make the test fail.